### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by CongMa13**

### CODE REVIEW

**Comment by CongMa13** [comment]
Hi @mkarunan @dlangbe 

Could you please check if there is any missing content that needs to be supplemented?

**Comment by mkarunan** [comment]
I guess it should also be targeted for release-staging-6.1 branch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,20 @@
...
+### Additions
+
+* API support
+  * Permutation of rank 4 tensors
+* Data type support
+  * Contraction - new supported data type f16, bf16, complex f32, complex f64
+
+### Fixes
+
+* Fixed bug in congtraction calculation with data type f32
```
Comment (mkarunan) [inline CHANGELOG.md]: contraction*

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,20 @@
...
-## (Unreleased) hipTensor 1.1.0 for ROCm 6.0.0
+## (Unreleased) hipTensor 1.2.0 for ROCm 6.1.0
+
+### Additions
+
+* API support
+  * Permutation of rank 4 tensors
+* Data type support
+  * Contraction - new supported data type f16, bf16, complex f32, complex f64
+
```
Comment (mkarunan) [inline CHANGELOG.md:16]: Added Scale and Bilinear Contraction samples and tests for complex float types.

Comment (mkarunan) [reply CHANGELOG.md:16]: Added Permutation Samples and Tests for F16, F32 types

Comment (CongMa13) [reply CHANGELOG.md:16]: done

File: CHANGELOG.md (hunk 1 above)
Comment (CongMa13) [reply CHANGELOG.md]: done

**Review by mkarunan — DISMISSED** [review]
LGTM

**Comment by CongMa13** [comment]
Hi @LisaDelaney , could you help review this PR?

**Comment by CongMa13** [comment]
Hi @LisaDelaney @saadrahim Could you help to review and approve this PR?

**Review by samjwu — DISMISSED** [review]

**Review by cgmillette — APPROVED** [review]

**Pull request closed without merging**